test/easy-stackup_test.js

```diff
@@ -0,0 +1,149 @@
+// test suite for simpler API
+'use strict'
+
+var expect = require('chai').expect
+
+var easyStackup = require('../lib/easy-stackup')
+
+var emptyGerber = 'G04 empty gerber*\nM02*\n'
+
+describe('easy-stackup function', function() {
+  it('should accept and call node style callback', function(done) {
+    easyStackup([], function(error, stackup) {
+      expect(error).to.not.be.ok
+      expect(stackup).to.be.ok
+      done()
+    })
+  })
```

Comment:
Can you please put newlines between the it blocks? Couldn't find an eslint rule for it, unfortunately